test/unit/pdf_find_controller_spec.js

```diff
@@ -1022,4 +1022,35 @@ describe("pdf_find_controller", function () {
       pageMatchesLength: [[1, 1, 1, 1, 1, 1, 1, 1, 1]],
     });
   });
+
+  it("dispatches updateFindControlState with picked state properties", async function () {
+    const testOnFind = ({ eventBus }) =>
+      new Promise(function (resolve) {
+        const eventState = {
+          source: this,
+          type: "",
+          query: "Foo",
+          caseSensitive: true,
+          entireWord: true,
+          findPrevious: false,
+          matchDiacritics: false,
+        };
+        eventBus.dispatch("find", eventState);
+
+        eventBus.on(
+          "updatefindcontrolstate",
+          function onUpdatefindcontrolstate(evt) {
+            expect(evt.state).toBe(FindState.NOT_FOUND);
+            expect(evt.previous).toBe(false);
+            expect(evt.entireWord).toBe(true);
+            expect(evt.matchesCount).toEqual({ current: 0, total: 0 });
+            expect(evt.rawQuery).toBe("Foo");
```

Comment:
Thanks, I can remove the function name, while the `toEqual` has problem that `evt` has additional members like `source` (which is an Object) and `type`, I suggest to not compare entire object since the extra members are not we interested most.

Reply:
In that case, wrapping the Object in `jasmine.objectContaining(...)` should work.

Reply:
Thank you, just updated with auto lint --fix.